cli/tests/unit/test_marathon_pod.py

```diff
@@ -155,13 +219,46 @@ def _assert_pod_list_propagates_exceptions_from_list_pod(exception):
     assert exception_info.value == exception
 
 
+@patch('dcoscli.marathon.main.emitter', autospec=True)
+def _assert_pod_update_invoked_successfully(
+        emitter, pod_id, properties, force, resource, deployment_id, emitted):
+    resource_reader = create_autospec(main.ResourceReader)
+    resource_reader.from_properties_or_stdin.return_value = resource
+    marathon_client = create_autospec(marathon.Client)
+    marathon_client.update_pod.return_value = deployment_id
+    subcmd = main.MarathonSubcommand(resource_reader, lambda: marathon_client)
+
+    returncode = subcmd.pod_update(pod_id, properties, force)
+
+    assert returncode == 0
+    marathon_client.show_pod.assert_called_with(pod_id)
+    resource_reader.from_properties_or_stdin.assert_called_with(properties)
+    marathon_client.update_pod.assert_called_with(
+        pod_id, pod_json=resource, force=force)
+    emitter.publish.assert_called_with(emitted)
+
+
+def _assert_pod_update_propagates_exception(
+        resource_reader, marathon_client, error_message):
+    subcmd = main.MarathonSubcommand(resource_reader, lambda: marathon_client)
+
+    with pytest.raises(DCOSException) as exception_info:
+        subcmd.pod_update(pod_id='foo', properties=[], force=False)
+
+    assert str(exception_info.value) == error_message
+
+
 def _unused_reader_fixture():
     marathon_client = create_autospec(marathon.Client)
-    subcmd = main.MarathonSubcommand(_unused_resource_reader,
+    subcmd = main.MarathonSubcommand(_unused_resource_reader(),
                                      lambda: marathon_client)
 
     return subcmd, marathon_client
 
 
-def _unused_resource_reader(path):
-    assert False, "should not be called"
+def _unused_resource_reader():
```

Comment:
Hmm, we're *using* it in tests to assert that it *isn't used* in feature code...should I rename it to `_failing_resource_reader()`?